config/middleware/receive-message/map-request-params.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const configVars = {};
+
+configVars.client = 'gambit-conversations';
+configVars.containerProperty = 'body';
+configVars.emojiStripParam = 'text';
+configVars.lowercaseParam = 'text';
+configVars.paramsMap = {
+  keyword: 'keyword',
+  broadcastId: 'broadcast_id',
+  text: 'incoming_message',
+  mediaUrl: 'incoming_image_url',
```

Comment:
Maybe I'm not following the issue. By adding the `campaignId` in the `map-request-params` config, the `campaignId` being sent in the POST request `body` gets parsed out and injected into the request object, like `req.campaignId`, which is essentially the same thing the [receive-message](https://github.com/DoSomething/gambit/blob/703fa6d59bde8db4f8ab47247c0e4e7e64ef4804/lib/middleware/receive-message.js) middleware is doing, which is why I mentioned the middleware is not required.